src/panels/config/entity_registry/show-dialog-entity-registry-detail.ts

```diff
@@ -0,0 +1,29 @@
+import { fireEvent } from "../../../common/dom/fire_event";
+import {
+  EntityRegistryEntry,
+  EntityRegistryEntryUpdateParams,
+} from "../../../data/entity_registry";
+
+const dialogTag = "dialog-entity-registry-detail";
+
+export interface EntityRegistryDetailDialogParams {
+  entry: EntityRegistryEntry;
+  updateEntry: (
+    updates: Partial<EntityRegistryEntryUpdateParams>
+  ) => Promise<unknown>;
+  removeEntry: () => Promise<boolean>;
+}
+
+export const loadEntityRegistryDetailDialog = () =>
+  import(/* webpackChunkName: "entity-registry-detail-dialog" */ "./dialog-entity-registry-detail");
+
+export const showEntityRegistryDetailDialog = (
+  element: HTMLElement,
+  systemLogDetailParams: EntityRegistryDetailDialogParams
+): void => {
+  fireEvent(element, "show-dialog", {
```

Comment:
New and improved way of doing dialogs.

CC @bramkragten

Reply:
🎉